src/tools/MappingTools.ts

```diff
@@ -699,13 +701,20 @@ export function mapVariableStatement(
     variable.setIsConst(true);
   }
 
+  if (
+    fileVariable.getFullText(source).search(' await ') > -1 ||
```

Comment:
Regular expressions are not allowed in ES5

Please take a look to [this](https://www.tutorialspoint.com/typescript/typescript_string_search.htm). We need to find an alternative.